values.schema.json

```diff
@@ -0,0 +1,328 @@
+
+
+    "title": "Values",
+    "type": "object",
+    "properties": {
+
+      "nameOverride": {
+        "type": "string",
+        "description": "String to partially override common.names.fullname template with a string (will prepend the release name)"
+      },
+      "fullnameOverride": {
+        "type": "string",
+        "description": "String to fully override common.names.fullname template with a string"
+      },
+      "imagePullSecrets": {
+        "type": "array",
+        "description": "Image pull secrets",
+        "items": {
+          "type": "object"
+        }
+      },
+
+      "core": {
+        "type": "object",
+        "description": "Core deployment values",
+        "allOf": [
+          {
+            "$ref": "#/$defs/commonOsrdService"
+          },
+          {
+            "properties": {
+              "internalUrl": {
+                "type": "string",
+                "description": "Internal URL for the service"
+              }
+            },
+            "required": ["internalUrl"]
+          }
+        ]
+      },
+
+      "editoast": {
+        "allOf": [
+            {
+                "$ref": "#/$defs/commonOsrdService"
+            },
+            {
+                "properties": {
+                    "init": {
+                        "properties": {
+                            "enabled": {
+                                "type": "boolean",
+                                "description": "Enable or disable the initialization"
+                            },
+                            "extend": {
+                                "type": "string",
+                                "description": "Extend the initialization with custom shell script"
+                            },
+                            "internalUrl": {
+                              "type": "string",
+                              "description": "Internal URL for the service"
+                            }
+                        }
+                    }
+                }
+            }
+        ]
+      },
+
+      "tileServer": {
+        "allOf": [
+          {
+            "$ref": "#/properties/editoast"
+          },
+          {
+            "properties": {
+              "hpa": {
+                "type": "object",
+                "properties": {
+                  "enabled": {
+                    "type": "boolean"
+                  },
+                  "minReplicas": {
+                    "type": "integer"
+                  },
+                  "maxReplicas": {
+                    "type": "integer"
+                  },
+                  "targetCPUUtilizationPercentage": {
+                    "type": "integer"
+                  }
+                },
+                "required": ["enabled", "minReplicas", "maxReplicas", "targetCPUUtilizationPercentage"]
+              }
+            }
+          }
+        ]
+      },
+
+      "gateway": {
+        "allOf": [
+        { "$ref": "#/$defs/commonOsrdService" },
```

Comment:
minor: Do you have some kind of formatter on this file? All the other occurrences of this are on 3 lines and this one is weirdly indented 🤷

Reply:
I didn't lint the file, my bad. Going to do it.